.github/compatibility-test-matrices/release-v7.6.x/ica-chain-a.json

```diff
@@ -0,0 +1,25 @@
+{
+  "chain-a": [
+    "release-v7.6.x"
+  ],
+  "chain-b": [
+    "v8.3.0",
+    "v8.2.0",
+    "v7.5.0",
+    "v7.4.0",
+    "v6.3.0",
```

Comment:
do we need v6 tags since its EOL?

Reply:
I put the tags for EoL releases just to check with compatibility with them. I think it makes sense that, even if those release lines are not supported anymore, that we do some basic compatibility check with the new releases that we make.